lib/client.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+const Promise = require("bluebird");
```

Comment:
@flovilmart how do you feel about it being a `devDependency` only? It makes some of the tests a little cleaner but could be foregone in the main library.